src/m365/spe/commands/containertype/containertype-list.ts

```diff
@@ -0,0 +1,68 @@
+import { Logger } from '../../../../cli/Logger.js';
+import config from '../../../../config.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { ClientSvcResponse, ClientSvcResponseContents, FormDigestInfo, spo } from '../../../../utils/spo.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+import { ContainerTypeProperties } from '../../ContainerTypeProperties.js';
+
+class SpeContainertypeListCommand extends SpoCommand {
+  private allContainerTypes?: ContainerTypeProperties[];
+
+  public get name(): string {
+    return commands.CONTAINERTYPE_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all Container Types';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['ContainerTypeId', 'ContainerTypeName', 'OwningApplicationId'];
```

Comment:
but if we check in the response the `ContainerTypeName` and `OwningApplicationId` don't exist in the response.

```
{
    "_ObjectType_": "Microsoft.Online.SharePoint.TenantAdministration.SPContainerTypeProperties",
    "ApplicationRedirectUrl": null,
    "AzureSubscriptionId": "/Guid(00000000-0000-0000-0000-000000000000)/",
    "ContainerTypeId": "/Guid(32793d51-870d-07da-113f-b586dee1c12a)/",
    "CreationDate": null,
    "DisplayName": "standard",
    "ExpiryDate": null,
    "IsBillingProfileRequired": true,
    "OwningAppId": "/Guid(1b3b8660-9a44-4a7c-9c02-657f3ff5d5ac)/",
    "OwningTenantId": "/Guid(00000000-0000-0000-0000-000000000000)/",
    "Region": null,
    "ResourceGroup": null,
    "SPContainerTypeBillingClassification": 0
  }
```

we should change to
```suggestion
    return ['ContainerTypeId', 'DisplayName', 'OwningAppId'];
```

Reply:
oh, how I made such a mistake! Thanks for spotting the mistake:)